api_app/helpers.py

```diff
@@ -128,17 +127,8 @@ def filter_analyzers(
     return cleaned_analyzer_list
 
 
-def get_binary(job_id, job_object=None):
-    if not job_object:
-        job_object = models.Job.object_by_job_id(job_id)
-    logger.info(f"getting binary for job_id {job_id}")
-    job_file = job_object.file
-    logger.info(f"got job_file {job_file} for job_id {job_id}")
-
-    binary = job_file.read()
-    return binary
-
-
 def generate_sha256(job_id):
-    binary = get_binary(job_id)
-    return hashlib.sha256(binary).hexdigest()
+    from api_app.models import Job
```

Comment:
I think it could make sense to have this function directly be part of `Job` model. Like:

```def get_binary_sha256(self) -> str:
       return hashlib.sha256(self.file.read()).hexdigest() 
```

Reply:
or,

```python3
@property
def sha256(self) -> str:
```

Reply:
Agree, i hate the `helpers.py` file. It should be better to add these functions in the correct models/file